lib/SymbolGraphGen/SymbolGraph.cpp

```diff
@@ -674,12 +674,6 @@ bool SymbolGraph::isUnconditionallyUnavailableOnAllPlatforms(const Decl *D) cons
 
 /// Returns `true` if the symbol should be included as a node in the graph.
 bool SymbolGraph::canIncludeDeclAsNode(const Decl *D) const {
-  // If this decl isn't in this module, don't record it,
-  // as it will appear elsewhere in its module's symbol graph.
-  if (D->getModuleContext()->getName() != M.getName()) {
```

Comment:
Something feels off about removing this change entirely. It's probably fine as-is, but i wonder if we could have the `SymbolGraph` type get the set of exported-import modules and have a version of `isFromExportedImportedModule` like you wrote on the AST Walker? That way we can be thorough.

Reply:
Yep, I agree. I added that check back with a `isFromExportedImportedModule`.